pages/dkp/kommander/2.2/install/small-environment/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: layout.pug
+navigationTitle: Install on small environment
+title: Install Kommander on a small environment
+menuWeight: 40
+excerpt: Install Kommander on a small environment
+beta: false
+enterprise: false
+---
+
+You can install Kommander on a small environment with smaller memory, storage, and CPU requirements for testing and demo purposes. This topic describes methods for installing Kommander in these environments. Refer to [Kommander documentation][2.2] for more information.
+
+You can install this minimal setup on networked and air-gapped environments, regardless of the license type (Essential or Enterprise).
+
+## Prerequisites
+
+Ensure you have done the following:
+
+- You have created a Kubernetes cluster.
+- You have acquired a DKP license.
+- You have installed Konvoy.
+- You have reviewed the prerequisite section pertaining to your [air-gapped][airgap], or [networked][networked] environment.
+
+## Minimal Kommander installation
+
+The `yaml` file that is used to install a minimal configuration of Kommander contains the bare minimum setup that allows you to deploy applications, access the DKP UI, create and attach clusters. It does **NOT** include applications for cost monitoring, logging, alerting, object storage, etc.
+
+In this `yaml` file you can find commented lines, that correspond to all platform applications which would be included in normal Kommander setup. Since these lines are commented, they are not taken into account during installation. If you want to test an additional platform application, you can enable it individually to be installed by uncommenting the corresponding line in the `yaml` file.
```

Comment:
```suggestion
In this `yaml` file you can find commented lines, that correspond to all platform applications which would be included in a normal Kommander setup. Since these lines are commented, they are not taken into account during installation. If you want to test an additional platform application, you can enable it individually to be installed by uncommenting the corresponding line in the `yaml` file.
```